integration/harness-e2e-cli/e2e/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "compilerOptions": {
+        "module": "es2020",
+        "target": "es2020",
+        "moduleResolution": "node",
+    }
+}
```

Comment:
Our other JSON files use 2 space indentation. Is there a reason that this specific file uses 4 space indentation?

Reply:
Just my new machine 😛 I want to add json formatting for prettier!